### ISSUE TITLE
Mastadon applet certified capacity slider prevents deployment 

### ISSUE THREAD

**Issue by Parkers145** [threefoldtech/www-mastodon#139]
### Description

During todays tech call while demonstrating the mastodon applet I found that the certified capacity slider prevented proper deployment. 

### Steps to reproduce

- Attempt to Create a deployment using all defaults with theeefoldconnect and certified capacity sliders set to on. 

- switch to advanced menu and expand the node selection section, only one node was shown, 39 which was unavailable.

- turning off the certified capacity showed all nodes in the selection dialog and allowed deployment, successful deployment 

### Logs/Alerts

https://youtu.be/7k1tvPgdXsk

**Comment by xmonader** [comment]
I don't believe that's an actual issue, the desired behavior was to deploy on certified nodes by default
